src/trivia/util.h

```diff
@@ -643,6 +643,14 @@ is_exp_of_two(unsigned n)
 void
 thread_sleep(double sec);
 
+/**
+ * Getenv wrapper returning the corresponding value only when it'd fit in a
+ * buffer of the given size. That is, its length should be <= buffer size - 1,
+ * because one extra char is taken by the trailing '\0'.
+ */
+char *
+getenv_safe(const char *name, size_t buf_size, bool *exceeded);
```

Comment:
How are we planning to enforce usage of this new function instead of `getenv` in the new code? Should we add a check to `checkpatch` like we did for [unsafe string functions](https://github.com/tarantool/checkpatch/blob/9479d77317693086e0ec321a22f983bc035a36a0/checkpatch.pl#L4921-L4938)?

Reply:
I think yes, a `checkpatch` rule should do

Reply:
> I think yes, a checkpatch rule should do

I'll add one once this PR lands into trunk.

Reply:
https://github.com/tarantool/checkpatch/pull/48